### PR TITLE
Handle lost session on rpc data

### PR DESCRIPTION
**Pull request by albinekb**
Solves #401, check if the session exists like when running `exit`

However, i think there's something that's not done correctly when closing tabs that makes this happen, will investigate more. Possibly race-condition?

If you open a couple of tabs and then close them all quickly it logs `session not found by` from `rpc.on('exit')..`, maybe the close is happening more than once if you do it fast, before it actually got time to close.

### CODE REVIEW

**Comment by rauchg** [comment]
There's definitely a race condition. I think we need to solve it properly. We should probably use a redux logger to inspect the state over time. The problem is that I was only able to reproduce this "at high speed". I would turn on the redux logger and then the problem would go away. 

tl;DR: https://en.wikipedia.org/wiki/Heisenbug
